common/state/src/lib.rs

```diff
@@ -23,6 +25,7 @@ pub use offsets::{Offset, OffsetMap};
 pub use span::{Span, SpanError, SpanVec};
 
 const OFFSET_NAME: &str = "file_offsets";
+const ROCKSDB_CACHE_SIZE: usize = 1024 * 500;
```

Comment:
config option? tunable

Reply:
I believe there is an additional ticket to expose more of these options via configuration. For this task, the scope was just reduce the default.